TulparAudio/internal/include/tulpar/internal/ListenerController.hpp

```diff
@@ -0,0 +1,64 @@
+/*
+* Copyright (C) 2018 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#ifndef TULPAR_INTERNAL_LISTENER_COLLECTION_HPP
+#define TULPAR_INTERNAL_LISTENER_COLLECTION_HPP
+
+#include <tulpar/audio/Listener.hpp>
+
+#include <mule/asset/Content.hpp>
+
+#include <array>
+#include <memory>
+
+namespace tulpar
+{
+namespace internal
+{
+
+/** @brief  Controller working with listener object */
+class ListenerController
```

Comment:
Should this class be replaced with free utility functions? All of its members can be static.